torch/nn/utils/rnn.py

```diff
@@ -30,14 +30,14 @@ class PackedSequence(PackedSequence_):
             information about the batch size at each sequence step
 
     """
-    def __new__(cls, *args):
+    def __new__(cls, data, batch_sizes=None):
```

Comment:
nit: Can you call this `data_or_pair`?

Reply:
Also, it would be good to leave a comment.

Reply:
We can't call it `data_or_pair` or the pattern `PackedSequence(data=x)` breaks. 

I've added some comments. Let me know if this looks better. :)